site/content/in-dev/unreleased/_index.md

```diff
@@ -102,8 +102,8 @@ nested namespaces. Iceberg tables belong to namespaces.
 
 ### Storage configuration
 
-A storage configuration stores a generated identity and access management (IAM) entity for your cloud storage and is created
-when you create a catalog. The storage configuration is used to set the values to connect Polaris to your cloud storage. During the
```

Comment:
nit: maybe remove "cloud" from https://polaris.apache.org/in-dev/unreleased/getting-started/creating-a-catalog/s3/ too?